CrossDisciplinary-1/UT-CrossDisciplinary-1.ifc

```diff
@@ -0,0 +1,49 @@
+#5 = IFCORGANIZATION($, 'ACCA software', $, $, $);
+#6 = IFCAPPLICATION(#5, '100', 'ACCA IfcInfraGenerator', 'ACCA IfcInfraGenerator');
+#7 = IFCPROJECT('3TkAlDBHHF$xq2dgZCH3rI', #1, 'Default Project', $, $, $, $, (#16), #8);
+#8 = IFCUNITASSIGNMENT((#9, #10, #11, #12));
+#9 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#10 = IFCSIUNIT(*, .AREAUNIT., $, .SQUARE_METRE.);
+#11 = IFCSIUNIT(*, .VOLUMEUNIT., $, .CUBIC_METRE.);
+#12 = IFCCONVERSIONBASEDUNIT(#13, .PLANEANGLEUNIT., 'DEGREE', #14);
+#13 = IFCDIMENSIONALEXPONENTS(0, 0, 0, 0, 0, 0, 0);
+#14 = IFCMEASUREWITHUNIT(IFCRATIOMEASURE(1.74532925E-2), #15);
+#15 = IFCSIUNIT(*, .PLANEANGLEUNIT., $, .RADIAN.);
+#16 = IFCGEOMETRICREPRESENTATIONCONTEXT($, 'Model', 3, 1.E-4, #17, #19);
+#17 = IFCAXIS2PLACEMENT3D(#18, $, $);
+#18 = IFCCARTESIANPOINT((0., 0., 0.));
+#19 = IFCDIRECTION((0., 1.));
+#20 = IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Body', 'Model', *, *, *, *, #16, $, .MODEL_VIEW., $);
+#21 = IFCSITE('3t2PewUcHA5fz9fmoiszqp', #1, 'Default Site', $, $, #22, $, $, .ELEMENT., (0, 0, 0), (0, 0, 0), 0., $, $);
+#22 = IFCLOCALPLACEMENT($, #25);
+#23 = IFCDIRECTION((0., 0., 1.));
+#24 = IFCDIRECTION((1., 0., 0.));
+#25 = IFCAXIS2PLACEMENT3D(#18, #23, #24);
+#26 = IFCROAD('0DN9cEmCPFUgDDXQbLxZvD', #1, 'a road', $, $, $, $, $, .ELEMENT.);
+#27 = IFCFACILITYPART('0yFg1D40940PDUfPFz$nBF', #1, 'a road part', $, $, $, $, $, .ELEMENT., IFCROADPARTTYPEENUM(.ROADSEGMENT.), .LONGITUDINAL.);
+#28 = IFCFACILITYPART('3r4b0myeL6UA$c7cgEphKD', #1, 'a road part lateral, left', $, $, $, $, $, .ELEMENT., IFCROADPARTTYPEENUM(.ROADSEGMENT.), .LATERAL.);
+#29 = IFCFACILITYPART('3xh0isp$n0MvRhHMOZm3Gk', #1, 'a road part lateral, right', $, $, $, $, $, .ELEMENT., IFCROADPARTTYPEENUM(.ROADSEGMENT.), .LATERAL.);
+#30 = IFCFACILITYPART('0$8P9TLaP4YBOMwGhnb2_4', #1, 'a road intersection part', $, $, $, $, $, .ELEMENT., IFCROADPARTTYPEENUM(.RAILWAYCROSSING.), .LATERAL.);
+#31 = IFCRELAGGREGATES('2BaK0$xkP6bxJT_huGqpQE', #1, 'Facility Container', 'Facility to Facility Parts(s)', #26, (#27, #28, #29, #30));
+#32 = IFCRAILWAY('1WV4LHx696d9NMU3EKsxnl', #1, 'a railway', $, $, $, $, $, .ELEMENT.);
+#33 = IFCFACILITYPART('1tBZz5I9X9xxVQUnDpblqx', #1, 'a railway part', $, $, $, $, $, .ELEMENT., IFCROADPARTTYPEENUM(.RAILWAYCROSSING.), .NOTDEFINED.);
+#34 = IFCRELAGGREGATES('3eUeDpbR58FuV5O7uWofv5', #1, 'Facility Container', 'Facility to Facility Parts(s)', #32, (#33));
+#35 = IFCBRIDGE('0lkLAjfTH4VvYnWYwPXxp7', #1, 'a bridge', $, $, $, $, $, .ELEMENT., .NOTDEFINED.);
+#36 = IFCRELAGGREGATES('2ocfYp2vT1kfdwG_pOCGW8', #1, 'Site Container', 'Site to Facilities', #21, (#26, #32, #35));
+#37 = IFCRELAGGREGATES('1$R2J$E6r598daBHeZ7A5a', #1, 'Project Container', 'Project to Spatial Element', #7, (#21));
+#38 = IFCRELINTERFERESELEMENTS('1KGxl_MJzCugAzAbkkkI95', #1, $, $, #35, #26, $, 'PassesOver', .U.);
+#39 = IFCRELINTERFERESELEMENTS('258z1l3OLB2AfN2jNMLr8D', #1, $, $, #35, #32, $, 'PassesOver', .U.);

```

Comment:
should one of these be 'PassesUnder' as if they both are passing over the bridge then they must be side by side??

Reply:
OK